project/src/com/google/daggerqueryui/index.html

```diff
@@ -66,17 +66,33 @@
                     <ul>
                         <li>
                             <b class="query-name">deps</b>
-                            <b>binding</b> - to see all nodes dependencies
+                            <b>binding</b> - to see all node dependencies,
+                        </li>
+                        <li>
+                            <b class="query-name">rdeps</b>
+                            <b>binding</b> - to see all nodes that depend on the binding,
                         </li>
                         <li>
                             <b class="query-name">allpaths</b>
-                            <b>source target</b> - to see all paths between source and target nodes
+                            <b>source target</b> - to see all paths between source and target nodes,
                         </li>
                         <li>
                             <b class="query-name">somepath</b>
-                            <b>source target</b> - to see some path between source and target nodes
+                            <b>source target</b> - to see some path between source and target nodes.
                         </li>
                     </ul>
+                    <p>
+                        To traverse an arbitrary graph use <b class="action-name">right-click</b> for showing node
+                        ancestors and <b class="action-name">left-click</b> for showing node dependencies.
+                        You can use the same actions to hide dependencies and ancestors as well.
+                    </p>
+                    <p>
+                        Consider the ability to simply <b class="action-name">specify the binding name</b>
+                        in the input field - if the graph contains such a node, it will be drawn on the plane.<br><br>
```

Comment:
Nit:
Plane is a bit of an uncommon word, would you mind changing it here & the instance below to something like screen, page, or canvas?

Reply:
Thank you! Let's replace it with **canvas**.